subsys/net/ip/Kconfig

```diff
@@ -551,6 +552,12 @@ config NET_CONTEXT_TXTIME
 	  should be sent. The TX time information should be placed into
 	  ancillary data field in sendmsg call.
 
+config NET_CONTEXT_RCVTIMEO
+	bool "Add RCVTIMEO support to net_context"
+	help
+	  It is possible to time out socket receive operation after configured
```

Comment:
I'd say "configured time" is ambiguous here: we're in Kconfig, so user seeing that will likely get question "where can I configure that time here?".

Something like: "Support setsockopt(SO_RCVTIMEO) call to set timeout for socket receive operations".

But formulated like that, makes us wonder if this option belongs to net/ip/Kconfig, or maybe net/lib/sockets/Kconfig (note that not I started it - you used the word "socket" yourself). The option still may be named generically, like "NET_OPT_RCVTIMEO".

Again, that's just a humble suggestion from a thorough review. Act only if you agree that would be an improvement. And I'd say, wait a possible nak from @jukkar, as he may have different considerations. I gave mine: if this option talks about "time out socket receive operation", then a user would expect to find it in socket configuration.

Reply:
I agree that "configured time" is ambiguous. However I think that this option should remain in IP Kconfig, alongside other options utilized by net context module. I'll rephrase help text to make it socket-less.